NGitLab/Impl/LintClient.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System.Net;
+using System.Threading;
+using System.Threading.Tasks;
+using NGitLab.Models;
+
+namespace NGitLab.Impl
+{
+    internal class LintClient : ILintClient
+    {
+        private readonly API _api;
+
+        public LintClient(API api)
+        {
+            _api = api;
+        }
+
+        public Task<LintCI> LintGitLabCIYamlAsync(string projectId, string yamlContent, LintCIOptions options, CancellationToken cancellationToken = default)
```

Comment:
I think the methods can have clearer names.

In [the doc](https://docs.gitlab.com/ee/api/lint.html#validate-a-ci-yaml-configuration-with-a-namespace), this is described as `Validate a CI YAML configuration with a namespace`, but to make it even clearer that we're passing in YAML content:

```suggestion
        public Task<LintCI> ValidateCIYamlContentAsync(string projectId, string yamlContent, LintCIOptions options, CancellationToken cancellationToken = default)
```